ports/stm32f4/supervisor/port.c

```diff
@@ -75,6 +69,6 @@ uint32_t port_get_saved_word(void) {
     return _ebss;
 }
 
-// void HardFault_Handler(void) {
+void HardFault_Handler(void) {
```

Comment:
If this is not provided by the HAL, you can put a `while (1) { }` here to hang when it happens. Makes debugging easier.